packages/account/src/Components/reset-trading-password-modal/reset-trading-password-modal.jsx

```diff
@@ -159,14 +159,7 @@ const ResetTradingPassword = ({ setDialogTitleFunc, toggleResetTradingPasswordMo
                                         </PasswordMeter>
                                     </fieldset>
                                     <Text as='p' size='xs' className='reset-trading-password__hint'>
-                                        {status.error_msg ? (
-                                            <Localize
-                                                i18n_default_text='{{error_msg}}'
-                                                values={{ error_msg: status.error_msg }}
-                                            />
-                                        ) : (
-                                            <Localize i18n_default_text='Strong passwords contain at least 8 characters that include uppercase and lowercase letters, numbers, and symbols.' />
-                                        )}
+                                        <Localize i18n_default_text='Strong passwords contain at least 8 characters that include uppercase and lowercase letters, numbers, and symbols.' />
```

Comment:
why are we removing this?

Reply:
Because the code block in line `163` is unreachable. In `line 126` the condition that renders the entire code block is !status.error_message`. But  in `line 162`, we have a condition to render an element only when the condition is `status.error_message`. This will never render

Reply:
Got it. Thanks